controlplane/kubeadm/config/manager/manager.yaml

```diff
@@ -21,7 +21,7 @@ spec:
         args:
         - "--leader-elect"
         - "--metrics-bind-addr=localhost:8080"
-        - "--feature-gates=ClusterTopology=${CLUSTER_TOPOLOGY:=false}"
+        - "--feature-gates=ClusterTopology=${CLUSTER_TOPOLOGY:=false},KubeadmIgnitionBootstrapFormat=${EXP_KUBEADM_IGNITION_BOOTSTRAP_FORMAT:=false}"
```

Comment:
Actually we *do* seem to need it here since this controller validates the `KubeadmConfigSpec` field. So, although the code which checks the feature flag lives under `bootstrap/kubeadm`, we need to enable the feature here, too.

Reply:
Good point! You're right

Reply:
This can be resolved.